src/lib/Libifl/auth.c

```diff
@@ -280,6 +280,10 @@ load_auths(int mode)
 	if (loaded_auths != NULL)
 		return 0;
 
+	if (pbs_conf.auth_method[0] == '\0' && pbs_conf.encrypt_method[0] == '\0') {
```

Comment:
Not sure but this looks wrong: because as per current pbs_conf's default value and pbs_loadconf(), pbs_conf.auth_method will never be the empty string, it will be either "resvport" (default value) or user/admin provided string. So if you are getting pbs_conf.auth_method[0] == '\0' then it will some bug somewhere in code.

Reply:
you're right, this was leftover for when i was doing testing. I'll remove it